src/system/SystemConfig.h

```diff
@@ -204,6 +208,20 @@
 #define CHIP_SYSTEM_CONFIG_FREERTOS_LOCKING 0
 #endif /* CHIP_SYSTEM_CONFIG_FREERTOS_LOCKING */
 
+/**
+ *  @def CHIP_SYSTEM_CONFIG_PLATFORM_LOCKING
+ *
+ *  @brief
+ *      Use PLATFORM locking.
```

Comment:
This should document what the requirements are or point to such documentation (e.g. what types/functions need to be defined, how they need to behave, etc).